akka-docs-dev/rst/scala/code/docs/stream/cookbook/RecipeKeepAlive.scala

```diff
@@ -22,16 +22,16 @@ class RecipeKeepAlive extends RecipeSpec {
       val sink = Sink(sub)
 
       //#inject-keepalive
-      val keepAliveStream: Source[ByteString, Unit] = ticks
+      val keepAliveStream: Flow[Tick, ByteString, Unit] = Flow[Tick]
         .conflate(seed = (tick) => keepaliveMessage)((msg, newTick) => msg)
 
       val graph = FlowGraph.closed() { implicit builder =>
         import FlowGraph.Implicits._
         val unfairMerge = builder.add(MergePreferred[ByteString](1))
 
-        dataStream ~> unfairMerge.preferred
         // If data is available then no keepalive is injected
-        keepAliveStream ~> unfairMerge ~> sink
+        dataStream ~> unfairMerge.preferred
+        ticks ~> keepAliveStream ~> unfairMerge ~> sink
```

Comment:
I changed it from source to flow so the type of it is not polluted with implementation/testing details, such as materialization value beeing `TestPublisher.Probe<Tick>`. This is for Java side, and I changed Scala side to be symmetrical.

Changing name to `keepAliveFlow` sounds good?